internal/changelogs/README.md

```diff
@@ -0,0 +1,19 @@
+## How to Prepare a Release for the Go SDK
+
+Step 1. Make any changes to the SDK as required
```

Comment:
Let's mention a few other things:
* script can be run from main and feature branch. If ran from main, a new branch will be created.
* one needs GitHub write permissions to the Go SDK repo to run this script

Reply:
Done!